src/modules/file-upload-input.marko

```diff
@@ -0,0 +1,84 @@
+<div id="file-upload-input">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>
+        The file-upload-input allows users to upload one or more files to a location.
+    </p>
+    <h4>Drag and Drop</h4>
+    <p>
+        Since a label element is used to achieve the UI aesthetic, the drag and drop interaction will need to be added with JS.
+    </p>
+
+    <h3>Base</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="file-upload-input">
+                <input
+                    id="file-upload-input__input"
```

Comment:
No need to use BEM syntax on an ID. Something like `file-upload-example-1` should be fine.